tests/ecmp/test_fgnhg.py

```diff
@@ -173,8 +178,94 @@ def partial_ptf_runner(ptfhost, test_case, dst_ip, exp_flow_count, **kwargs):
             qlen=1000,
             log_file=log_file)
 
+def hash_view_cli_verification(duthost, prefix_list):
```

Comment:
@kktheballer how does it correlate with the warm boot scenario?

Reply:
It is not correlated
It is kind of another supplementary test case